client/apps/employee/e2e/README.md

```
@@ -0,0 +1,275 @@
+# Run specific test file
+pnpm --filter @client/employee test:e2e login.spec.ts
+
+# Run tests on specific browser
+pnpm --filter @client/employee test:e2e --project=chromium
+pnpm --filter @client/employee test:e2e --project=firefox
+pnpm --filter @client/employee test:e2e --project="Mobile Chrome"
+```
+
+## 📁 Test Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)